ods_ci/run_robot_test.sh

```diff
@@ -362,7 +362,7 @@ if ! ${SUBFOLDER}; then
   case "$(uname -s)" in
     Darwin)
         # shellcheck disable=SC2046
-        TEST_ARTIFACT_DIR=$(mktemp -d  "${TEST_ARTIFACT_DIR}" -t "${TEST_ARTIFACT_DIR}"/ods-ci-$(date +%Y-%m-%d-%H-%M)-XXXXXXXXXX)
+        TEST_ARTIFACT_DIR=$(mktemp -d  -t ods-ci-$(date +%Y-%m-%d-%H-%M)-XXXXXXXXXX) -p "${TEST_ARTIFACT_DIR}"
```

Comment:
Since it is same as below, could you, please, at least unify the lines so it's obvious it's same or you can fallthrough (I would keep the `case` in case we'll realize that we need to differentate in the future again).

Reply:
Good catch, did not notice that 😄